components/servo/webview.rs

```diff
@@ -337,6 +337,7 @@ impl WebView {
         self.inner().rect
     }
 
+    /// Update Compositor's [`WebViewRenderer`] rect and notify repaint.
```

Comment:
I don't think we should talk about internal concepts like the "Compositor" in the public documentation. FWIW, the "Compositor" is misnamed as it doesn't work like a normal web engine compositor any longer, so the name will need to change eventually. I think for now we can omit the documentation and write it at a later time.